nifi-nar-bundles/nifi-update-attribute-bundle/nifi-update-attribute-ui/src/main/webapp/js/application.js

```diff
@@ -1598,9 +1635,9 @@ var ua = {
             // create the editor
             editor = $('<div></div>').addClass(editorClass).appendTo(wrapper).nfeditor({
                 languageId: languageId,
-                width: (cellNode.width() - 5) + 'px',
+                width: '250px',
```

Comment:
I think we want to retain the previous behavior where the editor is the width of the entire cell when it's opened.